extensions/wikia/VideosModule/Modules/VideosModuleStaff.class.php

```diff
@@ -38,7 +38,7 @@ public function getCacheKey() {
 		sort( $category );
 		$hashCategory = md5( json_encode( $category ) );
 
-		return implode( ':', $cacheKey, $hashCategory, $this->sort );
+		return implode( ':', [$cacheKey, $hashCategory, $this->sort] );
```

Comment:
Yup, works great :) The one last thing is whitespace. eg ```[$cacheKey, $hashCategory, $this->sort]``` should be ```[ $cacheKey, $hashCategory, $this->sort ]```. Same goes for the other changes as well.